src/main/java/com/onarandombox/MultiverseCore/MVWorld.java

```diff
@@ -516,22 +516,39 @@ public World getCBWorld() {
      */
     @Override
     public String getColoredWorldString() {
-        if (props.getAlias().length() == 0) {
+        if (!hasCustomName()) {
             props.setAlias(this.getName());
         }
 
-        if ((props.getColor() == null) || (props.getColor().getColor() == null)) {
+        if (!hasCustomColor()) {
             this.props.setColor(EnglishChatColor.WHITE);
         }
 
         StringBuilder nameBuilder = new StringBuilder().append(props.getColor().getColor());
-        if (props.getStyle().getColor() != null)
+        if (hasCustomStyle()) {
             nameBuilder.append(props.getStyle().getColor());
-        nameBuilder.append(props.getAlias()).append(ChatColor.WHITE).toString();
+        }
+        nameBuilder.append(props.getAlias()).append(ChatColor.WHITE);
 
         return nameBuilder.toString();
     }
 
+    private boolean hasCustomName() {
+        return props.getAlias() != null
+                && !props.getAlias().isEmpty()
+                && !props.getAlias().equals(name);
+    }
+
+    private boolean hasCustomColor() {
+        return props.getColor() != null
+                && props.getColor().getColor() != null
+                && !props.getColor().getColor().equals(ChatColor.WHITE);
```

Comment:
Use null-safe check when you trying to compare enums.
```suggestion
                && props.getColor().getColor() != ChatColor.WHITE;
```